### PR TITLE
I have add a testimonial

### PR DESCRIPTION
**Pull request by Terrocraft**
i have add a testimonial.

### CODE REVIEW

File: testimonials/testimonials.json
Comment (FllipEis) [inline testimonials/testimonials.json:1]: lgtm 👍

File: testimonials/skins/terro.png
Comment (FllipEis) [inline testimonials/skins/terro.png:1]: can you use a resolution of 16x16

**Review by FllipEis — CHANGES_REQUESTED** [review]

File: testimonials/skins/terro.png
Comment (Terrocraft) [reply testimonials/skins/terro.png:1]: Okay, waint

**Comment by Terrocraft** [comment]
Because I don't know how to do something in a pull request something Commite

**Comment by FllipEis** [comment]
Just commit your changes in the same branch. The PR will automatically update

**Comment by Terrocraft** [comment]
oh, thanks

**Pull request closed without merging**